### PR TITLE
Fix bug and support

### PR DESCRIPTION
**Pull request by dangvm-0369**
## Related Tickets:
- [#69796](https://edu-redmine.sun-asterisk.vn/issues/69796)

## WHAT this PR does?
- Fix support serial order
- Fix bug quantity product reduce when pay order
- Format view, icon 

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by dangvm-0369** [comment]
ready

File: app/Http/Controllers/BillController.php [hunk 1]
```diff
@@ -5,26 +5,91 @@
...
+        foreach ($order->products as $product) {
+            $quantityOrderProduct = $product->pivot->quantity;
+            $quantityProduct = $product->quantity;
+
+            if ($quantityProduct < $quantityOrderProduct) {
+                return redirect()->back()->withErrors(
+                    [
+                        'store' => __('Warning-Product-Quantity-Run-Out-Description', [
+                            'ProductName' => $product->name,
+                            'QuantityProduct' => $quantityProduct
```
Comment (ghost) [inline app/Http/Controllers/BillController.php]:
```suggestion
                            'QuantityProduct' => $quantityProduct
```
```suggestion
                            'QuantityProduct' => $quantityProduct,
```

File: app/Http/Controllers/BillController.php [hunk 2]
```diff
@@ -5,26 +5,91 @@
...
+                        DB::table('products')
+                            ->where('id', '=', $product->id)
+                            ->update([ 'quantity' => $updateQuantityProduct ]);
 
-        return redirect()->route('bills.show', [$order->id]);
+                        if ($quantityProduct == $quantityOrderProduct) {
+                            $title = __('Product-Out-Of-Stock');
+                            $message = __('Warning-Product-Quantity-Run-Out-Description', [
+                                'ProductName' => $product->name,
+                                'QuantityProduct' => $updateQuantityProduct
```
Comment (ghost) [inline app/Http/Controllers/BillController.php]:
```suggestion
                                'QuantityProduct' => $updateQuantityProduct,
```

File: app/Http/Controllers/OrderController.php [hunk 3]
```diff
@@ -239,22 +263,27 @@ function () use ($order) {
...
     {
-        $orders = Order::with(['customer', 'products'])->where('salon_id', session('selectedSalon'))
+        $orders = Order::with(['customer', 'products', 'bill'])->where('salon_id', session('selectedSalon'))
             ->whereDate('created_at', now()->toDateString())
-            ->orderByDesc('created_at')
             ->get();
-        $count = count($orders);
 
+        $i = 1;
         foreach ($orders as $order) {
```
Comment (ghost) [inline app/Http/Controllers/OrderController.php]:
```suggestion
        foreach ($orders as $index => $order) { // sử dụng index thay cho $i++...
```

**Comment by dangvm-0369** [comment]
ready

File: app/Http/Controllers/BillController.php (hunk 2 above)
Comment (ghost) [reply app/Http/Controllers/BillController.php]: chua fix

File: app/Notifications/QuantityProductNotification.php [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+     *
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        return (new MailMessage)
+                    ->line('The introduction to the notification.')
+                    ->action('Notification Action', url('/'))
+                    ->line('Thank you for using our application!');
```
Comment (ghost) [inline app/Notifications/QuantityProductNotification.php]: indent 4 spaces

**Comment by dangvm-0369** [comment]
ready

**Pull request merged**